airflow/scripts/git_scripts/fetch_repo_data.py

```diff
@@ -0,0 +1,58 @@
+import requests
+
+
+token = 'put_token_here'   # add your token
+
+
+def get_repo_data(owner, repo, token):
+
+    """ Fetch the data of the repository including stars, forks, watchers,
+     views and unique views(how many different users). views only work on repos you own"""
+
+    headers = {'Authorization': f'token {token}', 'Accept': 'application/vnd.github.v3+json'}
+
+    # Get repository information
```

Comment:
split it to two functions:
Get_repo()
Clean_repo_response() 
functions need to be short and easy to read.
+ like the comment above try to create a class to handle everything.